packages/sampleCommons/package.json

```diff
@@ -29,24 +33,57 @@
     "@mantine/notifications": "^7.16.0",
     "@mdx-js/loader": "^3.0.0",
     "@mdx-js/react": "^3.0.0",
+    "@next/mdx": "^15.1.7",
     "cookies-next": "^4.3.0",
     "file-system-db": "^2.1.0",
     "idb": "^8.0.0",
     "jose": "^4.15.9",
-    "mantine-react-table": "^2.0.0-beta.8",
+    "jsonpath-plus": "^10.3.0",
+    "mantine-react-table": "^2.0.0-beta.9",
     "next": "^15.1.7",
-    "sharp": "^0.33.5"
+    "react": "^18.3.1",
+    "react-dom": "18.3.1",
+    "sharp": "^0.33.5",
+    "swr": "^2.2.5"
   },
   "devDependencies": {
+    "@axe-core/react": "^4.10.1",
     "@gen3/toolsff": "file:../tools",
-    "@next/mdx": "^15.1.7",
+    "@next/eslint-plugin-next": "^15.2.1",
+    "@tailwindcss/aspect-ratio": "^0.4.2",
+    "@tailwindcss/forms": "^0.5.3",
+    "@tailwindcss/line-clamp": "^0.4.2",
+    "@tailwindcss/typography": "^0.5.7",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.2.0",
+    "@types/lodash": "^4.14.202",
     "@types/mdx": "^2.0.13",
     "@types/react": "^18.3.18",
     "@types/react-dom": "^18.3.5",
+    "@typescript-eslint/eslint-plugin": "^8.7.0",
```

Comment:
All of the eslint packages are defined in the root package.json since they are shared with the other packages in the mono repo. If we define them here, we will have to bump them here, and if we don't, there might be two different versions of the packages. It differs from the `commons-frontend-app`, but it might make things more complete here.